src/gamebase.cpp

```diff
@@ -112,6 +112,7 @@ int Game::Run()
 	while( IsRunning() )
 	{
 		start = Clock::now();
+        SDL_GetWindowSize(window, &windowSize.x, &windowSize.y);
```

Comment:
seems quick enough to do this every frame: https://github.com/libsdl-org/SDL/blob/f500c1476634d07ab9c532ad78d1dab2a93c7869/src/video/SDL_video.c#L2348

Reply:
```suggestion
		SDL_GetWindowSize(window, &windowSize.x, &windowSize.y);
```